tests/kernel/queue/src/test_queue_user.c

```diff
@@ -151,8 +172,10 @@ void test_queue_alloc_append_user(void)
 		zassert_true(qd != NULL, NULL);
 		zassert_equal(qd->data, i, NULL);
 	}
+
```

Comment:
can you remove these extra whitespace changes

Reply:
OK

Reply:
hi @andrewboie , I have corrected my mistakes, could you please review it again, thanks.